flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/testutils/DockerImageVersions.java

```diff
@@ -24,9 +24,9 @@
  */
 public class DockerImageVersions {
 
-    public static final String KAFKA = "confluentinc/cp-kafka:7.4.4";
+    public static final String KAFKA = "confluentinc/cp-kafka:7.7.1";
```

Comment:
CP 7.7 https://docs.confluent.io/platform/current/release-notes/index.html used Kafka 3.7 while in the pom file we only bump the dependency to 3.6.2.

We should either use CP 7.6 or also increase the used kafka version to 3.7.x to avoid suprises

Reply:
The kafka version here is used for server. The pom specifies the client. I wouldn't want to force synchronization as this is rather unrealistic in real life.

Reply:
Btw 3.7.x had a breaking interface change on the things that we hack to get transactions working. That's why I left it at 3.6.2.